src/text/2-package-subsections/irf.tex

```diff
@@ -1,4 +1,17 @@
 \subsection{gammapy.irf}
 \label{ssec:gammapy-irf}
 \todo{Fabio Pintore}
-IRF classes
+The {\it gammapy.irf} sub-package contains all classes and functionalities able to handle Instrument Response Functions (\irf) in a wide variety of formats. Usually, \irfs store instrument properties in the form of multi-dimension tables, with quantities expressed in terms of energy (true or reconstructed), off-axis angles, detector coordinates and so on. The main information stored in the common \gammaray \irfs are the effective area (Aeff), energy dispersion (Edisp), point spread function (PSF) and background (BKG). The {\it gammapy.irf} sub-package can open and access specific \irf extensions, interpolate and evaluate the quantities of interest on both energy and spatial axes, convert their format or units in different kinds, plot or write them into output files. In the following, we list the main sub-classes of the sub-package:
+
+\begin{itemize}
\item Effective area: \gammapy provides the class {\it EffectiveAreaTable2D} to manage Aeff, which is usually defined in terms of true energy and offset angle. The class functionalities offer the possibility to read from files or to create it from scratch. {\it EffectiveAreaTable2D} can also convert, interpolate, write, and evaluate the effective area for a given energy and offset angles, or even plot the multi-dimensional Aeff table.
```

Comment:
Any idea what those `^M` characters are? Are those added by your editor?

Reply:
It seems the editor added this weird characters. Not sure why as I don't see them in the editor, but I think I've finally removed them.